README.md

```diff
@@ -1,85 +1,131 @@
 # Ottehr
 
+The production-ready, open-source EHR.
+
+This monorepo contains code for the entire Ottehr telehealth platform.
+
 ## First Time Setup
 
-### Node
+### ESLint
```

Comment:
Do not need ESLint extension for initial run. Should be moved elsewhere, perhaps to a section about developing/editing.

Reply:
Yeah, I was thinking about that as well - probably move it to the bottom or add it to the contribution.md file instead.